operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_CPS.md

```diff
@@ -17,21 +17,19 @@ This page covers the appropriate contents for a BOS session template in order to
   - [`<ramroot>`](#ramroot)
   - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
 - [`root=` kernel parameter](#root-kernel-parameter)
+- [Boot set S3 parameters](#boot-set-s3-parameters)
 - [Example session template input file](#example-session-template-input-file)
 - [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
 
-## Boot set S3 parameters
+The Content Projection Service \(CPS\) is an optional provider for `rootfs` on compute nodes.
```

Comment:
Why are parentheses backslash-escaped throughout these markdown files? I don't see a technical reason for that. SAT docs do not do this, and they render find in GitHub and GitHub Pages (HTML). See here for example: https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/system_admin_toolkit/about_sat/Introduction_to_SAT.md

Reply:
I don't know why they were escaped. I removed all escapes of parentheses.